lib/glific/providers/balance_worker.ex

```diff
@@ -35,4 +42,25 @@ defmodule Glific.Jobs.BSPBalanceWorker do
 
     :ok
   end
+
+  @spec send_low_balance_notification(integer(), non_neg_integer()) :: nil | {:ok, any}
+  defp send_low_balance_notification(bsp_balance, organization_id) when bsp_balance < 1 do
+    ## We need to check if we have already sent this notification in last 24 hours
+    category = "low_bsp_balance"
+    time = Glific.go_back_time(24)
+
+    if MailLog.mail_sent_in_past_time?(category, time, organization_id) == false do
+      {:ok, _} =
+        Partners.organization(organization_id)
+        |> LowBalanceAlertMail.new_mail(bsp_balance)
+        |> Mailer.send(%{
+          category: category,
+          organization_id: organization_id
+        })
+    end
+
+    {:ok, "no email"}
```

Comment:
seems like we'll always send this as the result? why?

Reply:
Adding other stuff here:

1. we should ideally figure out and use google smtp just to simplify and standardize on one platform
2. glific-support@glific.org should be cc'ed on all emails, so mohit and team can keep track of and remind NGOs as needed